project/app/utils/complexity/squad_score.py

```diff
@@ -1,5 +1,9 @@
 import joblib
 import pandas as pd
+from os import path
+
+PICKLE_PATH = path.join(path.dirname(__file__), "", "MinMaxScaler.pkl")
+scaler = joblib.load(PICKLE_PATH)
```

Comment:
Good thinking on this refactor!

Reply:
yeah I noticed that the sklearn future warning was popping up on every request so I moved it to avoid loading it multiple times.